src/lib/Server/Output/ValueObjectVisitor/RestContent.php

```diff
@@ -112,105 +104,93 @@ public function visit(Visitor $visitor, Generator $generator, $data)
         $generator->endObjectElement('CurrentVersion');
 
         $generator->startObjectElement('Section');
-        $generator->startAttribute(
+        $generator->attribute(
             'href',
             $this->router->generate('ibexa.rest.load_section', ['sectionId' => $contentInfo->sectionId])
         );
-        $generator->endAttribute('href');
         $generator->endObjectElement('Section');
 
         // Main location will not exist if we're visiting the content draft
         if ($data->mainLocation !== null) {
             $generator->startObjectElement('MainLocation', 'Location');
-            $generator->startAttribute(
+            $generator->attribute(
                 'href',
                 $this->router->generate(
                     'ibexa.rest.load_location',
                     ['locationPath' => trim($mainLocation->pathString, '/')]
                 )
             );
-            $generator->endAttribute('href');
             $generator->endObjectElement('MainLocation');
         }
 
         $generator->startObjectElement('Locations', 'LocationList');
-        $generator->startAttribute(
+        $generator->attribute(
             'href',
             $this->router->generate(
                 'ibexa.rest.load_locations_for_content',
                 ['contentId' => $contentInfo->id]
             )
         );
-        $generator->endAttribute('href');
         $generator->endObjectElement('Locations');
 
         $generator->startObjectElement('Owner', 'User');
-        $generator->startAttribute(
+        $generator->attribute(
             'href',
             $this->router->generate('ibexa.rest.load_user', ['userId' => $contentInfo->ownerId])
         );
-        $generator->endAttribute('href');
         $generator->endObjectElement('Owner');
 
         // Modification date will not exist if we're visiting the content draft
         if ($contentInfo->modificationDate !== null) {
-            $generator->startValueElement(
+            $generator->valueElement(
                 'lastModificationDate',
                 $contentInfo->modificationDate->format('c')
             );
-            $generator->endValueElement('lastModificationDate');
         }
 
         // Published date will not exist if we're visiting the content draft
         if ($contentInfo->publishedDate !== null) {
-            $generator->startValueElement(
+            $generator->valueElement(
                 'publishedDate',
                 ($contentInfo->publishedDate !== null
                     ? $contentInfo->publishedDate->format('c')
                     : null)
             );
```

Comment:
The statement at the line 155 doesn't make sense - it's gonna be always not null at this point.
```suggestion
            $generator->valueElement('publishedDate', $contentInfo->publishedDate->format('c'));
```